stock_prices/stock_prices.py

```diff
@@ -2,14 +2,31 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    # define return profit variable
+    profit_list = []
+    # iterate through prices
+    for i in range(len(prices) - 1):
+        # check if current price is more than min(price) right of the current price
+        if prices[i] < prices[i + 1]:
+            profit_list.append(max(prices[i + 1:]) - prices[i])
```

Comment:
I'm surprised this passes the tests even though there is a test where the answer is -10. Good work though.